source/develop/release-management/features/virt/headless-vm.html.md

```diff
@@ -13,32 +13,32 @@ feature_status: Merged
+Support Headless VMs in oVirt by adding an option to run a VM without any graphical console, for cases in which a graphical console is not needed for accessing this VM. This Headless VM will run without graphical and Video devices.
 
 ### Owner
 
 *   Name: Sharon Gratch (sgratch)
 *   Email: <sgratch@redhat.com>
 
-### Benefit to Ovirt
+### Benefit to oVirt
 
-Before this feature was implemented, OVirt Engine always added at least one graphical console to a running VM. Although this is right for most use cases, some VMs are required to run without graphical consoles from the start or after the initial setup. The reason for that can be due to host limitations or according to VM usage requirements (e.g. Real-Time VM).
+Before this feature was implemented, oVirt Engine always added at least one graphical console to a running VM. Although this is right for most use cases, some VMs are required to run without graphical consoles from the start or after the initial setup. The reason for that can be due to host limitations or according to VM usage requirements (e.g. Real-Time VM).
 
-This feature add a flexible, very simple to manage and reversible option to start a Headless VM at any time via Engine and still keep an option to access the VM for setup or administrative purposes if needed.
+This feature adds a flexible, very simple to manage and reversible option to start a Headless VM at any time via Engine and still keep an option to access the VM for setup or administrative purposes if needed.
 
 ### Usage
 
 *   Setting a VM as Headless can be done to new or existed VM at any time and it is a reversible option, meaning that a Headless VM can be switched to non-Headless and vice versa.
 
 *   Setting the VM as Headless can be done via the UI by checking/un-checking a new "Headless Mode" checkbox field added to the Console tab of the VM dialog (in advanced mode). In case of a running VM, this configuration change will require a VM restart. 
 
-*   A Headless VM Template or Pool or Instance Type can also be created on the same way as a VM, so in case the user wants to create or edit a VM to become Headless, he can also choose a Template, Pool or Instance type which are configured as Headless and "inherit" this property for that specific VM.
+*   A Headless VM Template or Pool or Instance Type can also be created the same way as a VM, so in case the user wants to create or edit a VM to become Headless, he can also choose a Template, Pool or Instance type which are configured as Headless and "inherit" this property for that specific VM.
 
 *   Invoking a graphic console via the Engine is disabled for a Headless VM and so are all options in the "Console Options" dialog.
 
-*   In case of a new created VM, the user has two options to access this VM via Ovirt for managing:
+*   In case of a new created VM, the user has two options to access this VM via oVirt for managing:
```

Comment:
It is currently mentioned as a limitation since we decided not to handle it for 4.1 and let the user be responsible for managing it if required. We should consider handling it for 4.2.